courses/api.py

```diff
@@ -393,8 +393,15 @@ def deactivate_run_enrollment(
         order__purchaser=run_enrollment.user,
     )
     if line:
-        line_id = line.id
-        sync_line_item_with_hubspot(line_id)
+        line_id = line.first().id
+        try:
+            sync_line_item_with_hubspot(line_id)
```

Comment:
Another option would be to create an autoretrying task for this, like there is for deals, contacts, etc.  Or just call `hubspot_sync.tasks.sync_deal_with_hubspot` instead since that will also sync the lines.

```python
@app.task(
    acks_late=True,
    autoretry_for=(TooManyRequestsException, BlockingIOError),
    max_retries=3,
    retry_backoff=60,
    retry_jitter=True,
)
@raise_429
@single_task(10, key=task_obj_lock)
def sync_line_with_hubspot(line_id: int) -> str:
    """
    Sync a Line with a hubspot line

    Args:
        line_id(int): The Line id

    Returns:
        str: The hubspot id for the line
    """
    return api.sync_line_with_hubspot(line_id).id
```